server/services/version.spec.ts

```diff
@@ -0,0 +1,87 @@
+import { ObjectId } from 'mongodb'
+import { ApprovalStates } from '../models/Deployment'
+import '../utils/mockMongo'
+import VersionModel from '../models/Version'
+import { createVersion, findModelVersions, findVersionById, findVersionByName, serializedVersionFields } from './version'
+import UserModel from '../models/User'
+
+const modelId = new ObjectId
+const anotherModelId = new ObjectId
+
+const testVersion: any = {
+  model: modelId,
+  version: '1',
+  metadata: {},
+  built: false,
+  managerApproved: ApprovalStates.NoResponse,
+  reviewerApproved: ApprovalStates.NoResponse,
+  state: {},
+  logs: [],
+  createdAt: new Date(),
+  updatedAt: new Date()
+}
+
+const testVersion2: any = {
+  model: anotherModelId,
+  version: '1',
+  metadata: {},
+  built: true,
+  managerApproved: ApprovalStates.Accepted,
+  reviewerApproved: ApprovalStates.NoResponse,
+  state: {},
+  logs: [],
+  createdAt: new Date(),
+  updatedAt: new Date()
+}
+
+const testUser = {
+  userId: 'user1',
+  email: 'user1@email.com',
+  data: { some: 'value' },
+}
+const userDoc = new UserModel(testUser)
+
+describe('test version service', () => {
+
+  beforeEach(async () => {
+    const versionDoc = new VersionModel(testVersion)
```

Comment:
Again, `await VersionModel.create(testVersion)` is shorter.

Reply:
done